receiver/tlscheckreceiver/documentation.md

```diff
@@ -0,0 +1,59 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# tlscheckreceiver
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### tlscheck.days_left
+
+Days until X.509 certificate expiration.
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| s | Gauge | Int |
+
+### tlscheck.error
+
+Records errors occurring during HTTP check.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| {error} | Sum | Int | Cumulative | false |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| error.message | Error message recorded during check | Any Str |
+
+### tlscheck.issued_days
+
+Day duration of timespan certificate is issued for.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| s | Sum | Int | Cumulative | false |
```

Comment:
how do we reconcile the unit type of second with the value we show? Is the value 60 * 60 * 24 * days? If that is so then the metric of issued_days shouldn't have the unit type of `s`. Hmmm!